src/generators/AbstractGenerator.ts

```diff
@@ -1,16 +1,15 @@
-import { CommonInputModel, CommonModel, OutputModel, Preset, Presets } from '../models';
+import { CommonInputModel, CommonModel, OutputModel, Preset, Presets, RenderOutput, ProcessorOptions } from '../models';
 import { InputProcessor } from '../processors';
 import { IndentationTypes } from '../helpers';
 import { isPresetWithOptions } from '../utils';
-import { RenderOutput } from '../models/RenderOutput';
-
 export interface CommonGeneratorOptions<P extends Preset = Preset> {
   indentation?: {
     type: IndentationTypes;
     size: number;
   };
   defaultPreset?: P;
   presets?: Presets<P>;
+  processorOption?: ProcessorOptions
```

Comment:
Minor: does it make sense to rename the following?
```suggestion
  processorOptions?: ProcessorOptions
```

Reply:
yea, makes sense 👍